### PR TITLE
add definition for segments_acked and Alpha_aimd

### LINKED ISSUE #47: Question on AIMD-Friendly Region
Hi, while I am implementing this change in quiche, I want to make sure my understanding is correct on "AIMD-Friendly Region" section.

It says to use
```
W_{est} = W_{est} + α_{aimd} * \frac{segments\_acked}{cwnd}
```
To calculate W_est value and alpha_aimd initially is

```
α_{aimd} = 3 * \frac{1 - β_{cubic}}{1 + β_{cubic}}
```

Since β_{cubic is 0.7 (Section 4.6), it comes down to

```
α_{aimd} = 3 * (1-0.7)/(1+0.7) = 0.529
```

And α_{aimd} will become 1 when W_est >= W_max.

Which means in each ACK, W_est can be calculated as follows:

```
W_est = W_est + 0.529 * (segments_acked / cwnd)               (W_est < W_max)
W_est = W_est + 1 * (segments_acked / cwnd)                   (W_est >= W_max)
```

Is my understanding correct? My concern is when W_est < W_max, it's slower than Reno.

Also, I think the definition of segments_acked is missing in the draft.

### PR DESCRIPTION
**Pull request by goelvidhi**
Closes #47

### CODE REVIEW

**Review by junhochoi — APPROVED** [review]
LGTM

File: draft-eggert-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -423,6 +427,9 @@ that is, W<sub>cubic</sub>(*t* + *RTT*), as described in {{win-inc}}.
 An estimate for the congestion window in segments in the AIMD-friendly
 region, that is, an estimate for the congestion window of AIMD TCP.
 
+*segments_acked*:
+Number of segments acked when an ACK is received.
```
Comment (larseggert) [inline draft-eggert-tcpm-rfc8312bis.md:431]: `segments_acked` isn't actually used anywhere else in the document?

File: draft-eggert-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -553,7 +560,7 @@ W<sub>cubic</sub>(*t*) is less than *W<sub>est</sub>*. If so, CUBIC is
 in the AIMD-friendly region and *cwnd* SHOULD be set to
 *W<sub>est</sub>* at each reception of an ACK.
 
-*W<sub>est</sub>* is set equal to *cwnd* at the start of the
+*W<sub>est</sub>* is set equal to *cwnd<sub>start</sub>* at the start of the
```
Comment (larseggert) [inline draft-eggert-tcpm-rfc8312bis.md:563]: This seems like an unrelated change?

Comment (goelvidhi) [reply draft-eggert-tcpm-rfc8312bis.md:563]: It is. But I did it as it was a minor fix.

Comment (larseggert) [reply draft-eggert-tcpm-rfc8312bis.md:563]: OK

**Review by larseggert — APPROVED** [review]

**Comment by larseggert** [comment]
As this is minor, merging it now.

**Comment by larseggert** [comment]
I merged this and then realized there wasn't an entry for the changelog in the PR. @goelvidhi could you add that in your next PR?

**Comment by goelvidhi** [comment]
will do.
